vue-frontend/src/components/home-new/BlogSection.vue

```diff
@@ -0,0 +1,210 @@
+              >
+                <div
+                  class="tw-mt-[0.9375rem] tw-w-[4.5rem] 2xl:tw-w-[5rem] tw-text-black-core/[.87] tw-font-inter-medium tw-text-[0.625rem] md:tw-text-[0.875rem] 2xl:tw-text-[1.125rem] tw-leading-[0.8125rem] md:tw-leading-[1.125rem] 2xl:tw-leading-[1.463rem]"
+                >
+                  {{ blog.author }}
+                </div>
+                <div
+                  class="tw-mt-[0.9375rem] tw-w-[3.25rem] md:tw-w-[4.5rem] 2xl:tw-w-[5.938rem] tw-text-black-core/[.87] tw-font-inter-medium tw-text-[0.625rem] md:tw-text-[0.875rem] 2xl:tw-text-[1.125rem] tw-leading-[0.8125rem] md:tw-leading-[1.125rem] 2xl:tw-leading-[1.463rem]"
+                >
+                  Read more
+                </div>
+              </div>
+            </div>
+          </div>
+          <hr
+            v-if="
+              (blogs.length == 2 && index == 1) ||
+              (blogs.length == 3 && index == 2)
+            "
+            class="tw-hidden md:tw-block md:tw-ml-[4.375rem] tw-mt-[1.125rem] 2xl:tw-ml-[5.313rem] md:tw-w-[18.438rem] lg:tw-w-[27.5rem] 2xl:tw-w-[42.375rem] tw-h-[0.063rem] tw-bg-[#C6C6C6]"
+          />
+          <hr
+            v-else
+            class="tw-w-[17.5rem] md:tw-w-[18.438rem] lg:tw-w-[27.5rem] 2xl:tw-w-[42.375rem] tw-h-[0.063rem] tw-ml-[3.20rem] md:tw-ml-[4.375rem] 2xl:tw-ml-[5.313rem] tw-mt-[1.125rem] tw-bg-[#C6C6C6]"
+          />
+
+          <div
+            v-show="activeIndex === index"
+            class="tw-block md:!tw-hidden tw-ml-[3.125rem] tw-mt-[0.5rem] tw-mb-[0.5rem] tw-max-w-[17.625rem] tw-max-h-[10rem] tw-animate-fadeIn"
+          >
+            <a :href="blog.link">
+              <img
+                :src="blog.thumbnail"
+                class="tw-object-cover"
+                loading="lazy"
+                :alt="blog.title"
+              />
+            </a>
+            <hr
+              v-if="
+                (blogs.length == 2 && index == 0) ||
+                (blogs.length == 3 && index != 2)
+              "
+              class="md:tw-hidden tw-w-[17.5rem] md:tw-w-[18.438rem] lg:tw-w-[27.5rem] 2xl:tw-w-[42.375rem] tw-h-[0.063rem] tw-ml-[0.2rem] md:tw-ml-[4.375rem] 2xl:tw-ml-[5.313rem] tw-mt-[1.125rem] tw-bg-[#C6C6C6]"
+            />
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script type="module">
+import bgMobile from "@/assets/images/blog/bg/bg400.svg";
+import bg from "@/assets/images/blog/bg/bg2400.svg";
+import axios from "axios";
+import Config from "@/config.js";
+
+export default {
+  data() {
+    return {
+      width: 680,
+      bgMobile,
+      bg,
+      activeIndex: 0,
+      blogs: "",
+      blogsURL: Config.BLOG_URL,
+    };
+  },
+
+  mounted() {
+    this.getBlogs();
+    this.width = window.innerWidth;
+  },
+  methods: {
+    openLink() {
+      if (this.width < 680) {
+        return;
+      }
+      for (let i = 0; i < this.blogs.length; i++) {
```

Comment:
I think it will open all blogs, we should open only clicked blog. You can take ref from current blog section

Reply:
No it is opening clicked blog only

Reply:
Though this condition is not proper. If you see it, its running `for loop` on each blog, but we have clicked on one blog only. You can do it directly using `href="blog.link" target="_blank"`